src/Accounts/Authentication/Authentication.csproj

```diff
@@ -12,11 +12,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.11.2" />
+    <PackageReference Include="Azure.Identity" Version="1.12.0-beta.3" />
```

Comment:
Is the `beta` version stable for us to consume? As Identity looks like a key dependency for us. We might prefer a stable version.

Reply:
https://www.nuget.org/packages/Azure.Identity/1.12.0
We should be using the GA version